src/pages/menuUpload/itemMaster/ItemMasterPage.tsx

```diff
@@ -31,32 +45,133 @@ function ItemMasterPage() {
     dispatch(getItemRequest(formValues as any));
```

Comment:
Here `formValues` I think has `GetItemRequestPayload` type so `formValues as GetItemRequestPayload` should work